src/EventStore.Core/Cluster/Settings/ClusterVNodeSettings.cs

```diff
@@ -176,6 +180,9 @@ public class ClusterVNodeSettings {
 			Ensure.Positive(initializationThreads, "initializationThreads");
 			Ensure.NotNull(gossipAdvertiseInfo, "gossipAdvertiseInfo");
 
+			if (scavengeThrottlePercent <= 0 || scavengeThrottlePercent > 100)
```

Comment:
should backend cache size also be validated ?
`Ensure.NonNegative(scavengeBackendCacheSize, "scavengeBackendCacheSize");`

Reply:
I made it Ensure.Positive